.dotnet/src/Custom/Assistants/AssistantClient.cs

```diff
@@ -3,13 +3,15 @@
 using System.ClientModel;
 using System.ClientModel.Primitives;
 using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
 using System.Threading.Tasks;
 
 namespace OpenAI.Assistants;
 
 /// <summary>
 /// The service client for OpenAI assistants.
 /// </summary>
+[Experimental("AssistantsBetaV1")]
```

Comment:
@terrajobst, do you have conventions for naming experimental features?

Reply:
What you're naming there isn't the feature but the diagnostic ID that is being reported by the compiler. I recently helped to create a topic that explains their naming convention [here](https://learn.microsoft.com/en-us/dotnet/csharp/roslyn-sdk/choosing-diagnostic-ids).

Reply:
Thanks for that, I had previously looked for a document like this and wished it had been there!

I'll use `OPENAI001` as a more conforming replacement.